src/pages/Documentation.js

```diff
@@ -1,56 +1,135 @@
 import React from "react";
-import { Divider } from "../components";
-// import FileCopyIcon from "@material-ui/icons/FileCopy";
 import "../App.css";
+import { Divider } from "../components";
+import Sandbox from "../Sandbox";
+import data from '../data/example-data.json';
+import { 
+  ExpansionPanel, 
+  ExpansionPanelSummary, 
+  ExpansionPanelDetails 
+} from '@material-ui/core'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+
+const panelStyle = {
+  background: 'black',
+  color: 'white',
+};
+
+const expandIcon = <ExpandMoreIcon style={{color: 'white'}} />
 
 const Documentation = () => (
   <>
     <section className="endpoints">
       <h3>Endpoints</h3>
       <p>stranger-things-api.herokuapp.com/</p>
-      <ul className="endpoints-list">
-        <li>
-          <span className="bold">GET all characters</span>{" "}
-          <p>api/v1/characters</p> {/* <FileCopyIcon /> */}
-        </li>
-        <li>
-          <span className="bold">GET by ID</span> <p>api/v1/characters/:id</p>
-        </li>
-        <li>
-          <span className="bold">GET by count</span>{" "}
-          <p>api/v1/characters/random?count=1</p>
-        </li>
-        <li>
-          <span className="bold">GET by name</span>{" "}
-          <p>api/v1/characters?name=eleven</p>
-        </li>
-        <li>
-          <span className="bold">GET by query</span>{" "}
-          <p>api/v1/characters?hairColor=brown</p>
-        </li>
-        <li>
-          <span className="bold">GET all with pagination</span>{" "}
-          <p>api/v1/characters?perPage=20?page=1</p>
-        </li>
-      </ul>
-      <h3>Queries</h3>
-      <p>
-        aliases, otherRelations, affiliation, occupation, residence,
-        appearsInEpisodes, status, gender, eyeColor, born, hairColor,
-        portrayedBy
-      </p>
+      <ExpansionPanel
+        classes="endpoints-list"
+        style={panelStyle}
+      >
+        <ExpansionPanelSummary
+          expandIcon={expandIcon}
+          aria-controls="panel1a-content"
+          id="panel1a-header"
+          classes="endpoints-list"
+        >
+          <span className="bold">GET all characters</span> - /api/v1/characters
```

Comment:
How do you feel about just having a wide space here instead of the ` - `?
```suggestion
          <span className="bold">GET all characters</span>    /api/v1/characters
```